server/service/src/activity_log.rs

```diff
@@ -63,6 +63,34 @@ pub fn activity_log_entry(
     Ok(())
 }
 
+/// Use instead of `activity_log_entry()` to provide the store_id if the ctx doesn't include it (e.g. system or basic context used in changelog processors)
+pub fn activity_log_entry_with_store(
```

Comment:
I left the old method name alone to avoid updating a bajillion files. This name seems clear enough including the doc comment?